src/modules/lesson/controllers/lesson.controller.spec.ts

```diff
@@ -116,29 +117,39 @@ describe('LessonController', () => {
 
     it('success', async () => {
       lessonService.updateLesson.mockReturnValue(lessonEntity);
-      lessonService.updateLessonHashtag.mockReturnValue(lessonHashtag);
 
       const returnValue = await lessonController.updateLesson(
         param,
         updateLessonDto,
         memberId,
       );
 
+      expect(prismaHelper.validateOwnerOrFail).toBeCalledTimes(1);
       expect(lessonService.updateLesson).toBeCalledTimes(1);
-      expect(lessonService.updateLessonHashtag).toBeCalledTimes(1);
       expect(returnValue).toBeInstanceOf(LessonEntity);
     });
+  });
 
-    it('false - 과제 출제자가 아닌 사람이 수정을 하려고 했을 때', async () => {
-      lessonService.updateLesson.mockImplementation(() => {
-        throw new ForbiddenException('과제를 수정할 권한이 없습니다.');
-      });
-
-      await expect(async () => {
-        await lessonController.updateLesson(param, updateLessonDto, memberId);
-      }).rejects.toThrowError(
-        new ForbiddenException('과제를 수정할 권한이 없습니다.'),
+  describe('deleteLesson', () => {
+    let param: IdRequestParamDto;
+    let memberId: any;
+    let mockLesson: Omit<LessonEntity, 'hashtag'>;
+    beforeEach(async () => {
```

Comment:
```suggestion
    let mockLesson: Omit<LessonEntity, 'hashtag'>;
    
    beforeEach(() => {
```
구문간 \n 해주시고 필요없는 async 는 지워주세요

Reply:
넵 모든 파일에 필요없는 async 제거 및 구문 간 문단 나누기를 해주었습니다!